perllib/FixMyStreet/Cobrand/Oxfordshire.pm

```diff
@@ -160,6 +160,8 @@ sub open311_pre_send {
 
 sub on_map_default_status { return 'open'; }
 
+sub on_map_default_body_user_status { return ('confirmed', 'investigating', 'action scheduled'); }
```

Comment:
Rather than having a separate `on_map_default_body_user_status` function, would it be easier to in `Reports.pm` to set `$filter_status{$state} = 1` for each open state if $status{open} is set (and similar)? Then all the relevant options would be selected in the form, whether it was body user or not.

Reply:
Oxfordshire though, AIUI, don't want to include 'in progress' as part of the default but that is an open state hence this.

Reply:
But it won't appear, will it? The states that appear come from their `state_groups_inspect` function. So sure, `$filter_status{in progress}` will be 1, but there won't ever be a checkbox to select using that.